lib/rgpg/gpg_helper.rb

```diff
@@ -21,13 +21,19 @@ def self.generate_key_pair(key_base_name, recipient, real_name)
       end
     end
 
-    def self.encrypt_file(public_key_file_name, input_file_name, output_file_name)
+    def self.encrypt_file(public_key_file_name, input_file_name, output_file_name, cipher_algo: nil, digest_algo: nil, compress_algo: nil)
```

Comment:
How about just making this a double splat like `**options` or `options = {}`, then we can transparently pass whatever options we need without modifying this code again.  The leading `--` can be added in the `run_gpg_capture` arg list with something like
```ruby
*options.map { |k, v| ["--#{k.to_s.gsub('_', '-')}", v] }.flatten
```